README.md

```diff
@@ -53,11 +53,11 @@ This "Hello World!" example is built with the `0.9.0-SNAPSHOT` version of `finch
 
 ```scala
 import io.finch._
-import com.twitter.finagle.Http
```

Comment:
Great news !! ... so maybe change line 52 to mention the version "0.9.1-SNAPSHOT" (with the new Finagle 6.30.0) instead of the "0.9.0-SNAPSHOT" for the "Hello World!" sample.